src/SourceBuild/tarball/content/src/source-mappings.jsonc

```
@@ -1,7 +1,18 @@
+// This file configures where the VMR synchronizes the sources from.
+// Each individual repository has its own mapping which says where the remote repo is,
+// what files are excluded from the sync, etc.
+// More details can be found at: {{ TODO }}
```

This will be filled in once we have the docs from `dotnet/product-construction` somewhere public.

---

Is there maybe a GitHub issue that it could link to in the meantime?

---

Good point, I've created an issue for this

```suggestion
// More details can be found at: {{ TODO: https://github.com/dotnet/arcade/issues/11226 }}
```